### DISCUSSION TITLE
Ability to remove variable from using experssion

### DISCUSSION THREAD

**Discussion by HakamFostok** [dotnet/csharplang#7644] [Language Ideas] [answered]
This code compiles

```
using (StartObservabilityActivity())
{
	// everything is intendent
}
```

but this code does NOT compile

```
using StartObservabilityActivity();
// remaining of the scope
```

I am suggesting to allow the second code

### Motivation

the motivation behind this is: that since .NET 6, the observability started to take momentum and now there is a lot of creating [`Activity`](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.activity?view=net-7.0) which only need to be created and disposed, but nothing else (in most cases).
This will prevent the unnecessary indentation of the whole code that need to be observed.
Otherwise, we have to set it to the `_` variable to be able to use the using expression.

**Comment by HaloFour** [comment, marked as answer]
See: #2235
